test/intl402/NumberFormat/default-currency-maximum-fraction-digits.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 the V8 project authors. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+esid: ECMA-402 #sec-setnfdigitoptions
+description: >
+    When a currency is used in Intl.NumberFormat and minimumFractionDigits is
+    not provided, maximumFractionDigits should be range-checked against it.
+include: [assert.js]
+---*/
+
+assert.throws(RangeError,
+              () => new Intl.NumberFormat('en', {
+                  style: 'currency',
+                  currency: 'USD',
```

Comment:
I'm assuming __ISO 4217__ defines the currency digits to 2. It would be good to add another assertion in this file using some currency without a defined digits to get the default value 2 from [currencydigits](https://tc39.github.io/ecma402/#sec-currencydigits).